integration/tests/brig/user/test_client.py

```diff
@@ -0,0 +1,39 @@
+from helpers import setup
+import json
+
+def test_cant_delete_lh_client(ctx, prekeys):
+    uid = setup.random_user(ctx)['id']
+
+    with ctx.add_client(uid, [prekeys.pop()], prekeys.lpop(), 
+                        ctype="legalhold", internal=True) as r:
+        assert r.status_code == 201
+        resp = r.json()
+    lh_client_id = resp['id']
+
+    with ctx.delete_client(uid, lh_client_id) as r:
+        assert r.status_code == 404
```

Comment:
we get a 404 here because there is a bug in `delete_client()`: the path is wrong